src/ipaperftest/providers/vagrant.py

```diff
@@ -56,7 +56,7 @@ def generate_metadata(self, ctx, machine_configs, domain):
                 "cpus": 4,
             },
             "client": {
-                "memory": 1024,
+                "memory": 2048,
```

Comment:
Is this a temporary change?

Reply:
It's actually something I missed when working on the multiple providers change. With less memory commands start to fail as the client runs out of memory.

Reply:
Better to move it to a different commit? I'm ok with it in the same PR but some additional reasoning would be helpful for future devs.

Reply:
You are right, I moved it to a different commit.